gpbft/chain.go

```diff
@@ -22,6 +22,9 @@ const (
 	CidMaxLen = 38
 	// ChainMaxLen specifies the maximum length of a chain value.
 	ChainMaxLen = 100
+
+	// ChainDefaultLen specifies the default length of chain value.
+	ChainDefaultLen = 100
```

Comment:
Why introduce a new constant?

Reply:
Got it. Next question: Why the increase in max chain length from 100 to 128?

Reply:
done

Reply:
Because:
1. it aligns with the size of the merkle tree
2. if in future we want to increase it, the context was fresh in my head, so removing the techdebt of Max = Default was easy

But with intellectual honesty, it was first 1. and 2. is a bit more of post factum justification

Reply:
That makes sense to me. Thank you for documenting that in this comment. I updated the issue with it.